scripts/test-ts-definition-files.ts

```diff
@@ -112,12 +122,19 @@ function testProject(folder: string) {
   );
 
   const tarLocation = path.relative(projectLocation, tarPath);
-  console.log('===> Installing Puppeteer from tar file');
+  console.log('===> Clearing left over node_modules to ensure clean slate');
+  // We don't care if this errors because if it did it's most likely because
+  // there was no node_modules folder, which is fine.
+  spawnSync('rm', ['-r', path.join(projectLocation, 'node_modules')], {
```

Comment:
We could also use `fs.rmSync` with the `recursive` option, which seems less indirect. WDYT?

Q: What's `encoding` doing here?

Reply:
Ah nice idea 👍 

`encoding` was left over from refactor, as we don't need the output we don't need to care about it